lessons/Data-Science/filtering-and-building/langs/en-us/index.adoc

```diff
@@ -127,18 +127,15 @@ Let's think about what filtering and building would look like with Animals at th
 - Match the code on @printable-exercise{what-table-do-we-get-matching.adoc} with the descriptions of the tables they would produce.
 }
 
-@teacher{@opt{Give students some scaffolded practice filtering and building with the Animals! +
+@teacher{
+@opt{Give students some scaffolded practice filtering and building with the Animals! +
 
-- Practice using the Design Recipe to write new helper functions:   
-  * @opt-printable-exercise{design-recipe-1.adoc}
-  * @opt-printable-exercise{design-recipe-2.adoc}.
 - Practice using Circles of Evaluation with Filtering and Building to make:
   * @opt-printable-exercise{coe-table-old-cats.adoc, a Table of Old Cats} 
   * @opt-printable-exercise{coe-box-plot-young-dog-kilos.adoc, a Box Plot showing the Distribution of the weight (in kilos) of Young Dogs}
   * @opt-printable-exercise{coe-scatterplot-old-dogs.adoc, a Scatter Plot showing the Relationship between Kilograms and Weeks to adoption for Old Dogs}
-
-All of these Design Recipes are the function form of expressions from the @lesson-link{/lookups/pages/lookup-expressions.adoc} workbook page. Have students refer back to this page if they need support!
-}}
+}
+}
 
 @lesson-instruction{
 - Then turn to @printable-exercise{putting-it-all-together.adoc} to practice defining new tables, defining new helper functions, and building visualizations in the @starter-file{putting-it-all-together}.
```

Comment:
similarly, this page has a question about building a box plot

Reply:
fixed